web-console/4.0-designs/cluster-settings/cluster-settings.md

```diff
@@ -38,38 +38,36 @@
 
 ### Update Status
 
-**Up to Date**
-![1](img/overview-up-to-date.png)
+The update status will be displayed in the box area beneath the tabs. Update information will be shown in both the `Current Version` box and the `Update Status` box.
 
-- For clusters on the latest version, the pficon-ok will be displayed to the left of the "Up to date" status.
+**Up to Date**
+![1](img/up-to-date.png)
 
-**Update in Progress**
-![2](img/overivew-updating.png)
+- For clusters on the latest version, the update status will be `Up to date` with a green pficon-ok displayed to the left.
 
-- For clusters on currently going through an update, the pficon-in-progress will be displayed to the left of the "Updating" status.
-- An informational inline notification will also appear at the top of the page telling the user the update is in progress.
-- The link at the end of the inline notification will bring users to the Cluster Operators tab where they can monitor progress more closely. 
+**Update Available**
+![2](img/update-available.png)
```

Comment:
This is a nit but it seems like the update now button is really close to the status above it. Can we add a little more padding between them?